hub/management/commands/import_red_blue_wall_constituencies.py

```diff
@@ -26,8 +26,8 @@ class Command(BaseImportFromDataFrameCommand):
         "table": "areadata",
         "is_filterable": True,
         "options": [
-            dict(title="Red Wall", shader="red"),
-            dict(title="Blue Wall", shader="blue"),
+            dict(title="Red Wall", shader="red-500"),
+            dict(title="Blue Wall", shader="blue-500"),
```

Comment:
I must have something missing on my side because these both show up as black.

Reply:
This is in part because I omitted a dictionary lookup that allowed this to work. I have added that back in. Hopefully, this will work better, now